src/utils/save-file.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2022, Nuance, Inc. and its contributors.
+ * All rights reserved.
+ *
+ * This source code is licensed under the Apache-2.0 license found in
+ * the LICENSE file in the root directory of this source tree.
+ */
+
+/** eslint-disable unicorn/prefer-node-protocol */
```

Comment:
I think this comment to disable that eslint rule is no longer necessary as we are using the "node protocol", i.e. we are importing from "node:fs" and not "fs".